doc/man/8/mount.ceph.rst

```diff
@@ -1,182 +1,216 @@
+
+:command:`snapdirname`
+    string, set the name of the hidden snapdir. Default: .snap
 
-  - ``no``: never attempt to reconnect when client detects that it has been blacklisted. Blacklisted clients will not attempt to reconnect and their operations will fail too.
+:command:`write_congestion_kb`
+    int (kb), max writeback in flight. scale with available
+    memory. Default: calculated from available memory
 
-  - ``clean``: client reconnects to the ceph cluster automatically when it detects that it has been blacklisted. During reconnect, client drops dirty data/metadata, invalidates page caches and writable file handles.  After reconnect, file locks become stale because the MDS loses track of them. If an inode contains any stale file locks, read/write on the inode is not allowed until applications release all stale file locks.
+:command:`wsize`
+    int (bytes), max write size. Default: 16777216 (16*1024*1024) (writeback
+    uses smaller of wsize and stripe unit)
 
-Mount Secrets
-=============
-If the `secret` and `secretfile` options are not specified on the command-line
-then the mount helper will spawn a child process that will use the standard
-ceph library routines to find a keyring and fetch the secret from it.
 
 Examples
 ========
 
 Mount the full file system::
 
-        mount.ceph monhost:/ /mnt/foo
+    mount.ceph :/ /mnt/mycephfs
+
+Assuming mount.ceph is installed properly, it should be automatically invoked
+by mount(8)::
 
-If there are multiple monitors::
+    mount -t ceph :/ /mnt/mycephfs
 
-        mount.ceph monhost1,monhost2,monhost3:/ /mnt/foo
+Mount only part of the namespace/file system::
 
-If :doc:`ceph-mon <ceph-mon>`\(8) is running on a non-standard
-port::
+    mount.ceph :/some/directory/in/cephfs /mnt/mycephfs
 
-        mount.ceph monhost1:7000,monhost2:7000,monhost3:7000:/ /mnt/foo
+Mount non-default FS, in case cluster has multiple FSs::
 
-To automatically determine the monitor addresses from local configuration::
+    mount -t ceph :/ /mnt/mycephfs2 -o mds_namespace=mycephfs2
 
-        mount.ceph :/ /mnt/foo
+Pass the monitor host's IP address, optionally::
 
-To mount only part of the namespace::
+    mount.ceph 192.168.0.1:/ /mnt/mycephfs
 
-        mount.ceph monhost1:/some/small/thing /mnt/thing
+Pass the port along with IP address if it's running on a non-standard port::
 
-Assuming mount.ceph(8) is installed properly, it should be
-automatically invoked by mount(8) like so::
+    mount.ceph 192.168.0.1:7000:/ /mnt/mycephfs
 
-        mount -t ceph monhost:/ /mnt/foo
+If there are multiple monitors, passes addresses separated by a comma::
+
+   mount.ceph 192.168.0.1,192.168.0.2,192.168.0.3:/ /mnt/mycephfs
+
+If authentication is enabled on Ceph cluster::
+
+    mount.ceph :/ /mnt/mycephfs -o name=fs_username
+
+Pass secret key for CephX user optionally::
+
+    mount.ceph /mnt/mycephfs -o name=fs_username,secret=AQATSKdNGBnwLhAAnNDKnH65FmVKpXZJVasUeQ==
```

Comment:
The above command looks incorrect. You need a device string of some sort. I'd suggest just using `:/` except in the examples where you're showing how to pass in alternate info.

Reply:
Fixed.